lib/AST/ASTMangler.cpp

```diff
@@ -82,6 +82,10 @@ static StringRef getCodeForAccessorKind(AccessorKind kind) {
   switch (kind) {
   case AccessorKind::Get:
     return "g";
+  case AccessorKind::DistributedGet:
+    // TODO(distributed): probably does not matter since we mangle distributed
+    //  thunk getters as the name of the Storage?
+    return "g";
```

Comment:
TODO: iffy if we should mangle it specially - concerned that'll mess up backwards compat as well. 🤔 

Will think more about it; doing a "gd" is possible, but needs unmangling work and I think we'd need to then keep the old support because old concrete var calls would be with `g` already

Reply:
Won't we have a problem finding identifiers then because existing methods are mangled as `g` I guess? So we need to be careful here?

Reply:
But yeah probably worth it

Reply:
Might be reasonable to break now to avoid future problems.

Reply:
That is true to be honest. And 6 is a good time to do it...

I'd also want to perhaps then break the "we include class/struct" info etc... 🤔

Reply:
rdar://125340714